test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -42,6 +44,38 @@ struct UcxAmDesc {
     const ucp_am_recv_param_t    *_param;
 };
 
+void* operator new(std::size_t size)
+{
+    void *ptr;
+
+    ptr = malloc(size);
+    ucs_memtrack_allocated(ptr, size, "io-demo operator new");
+
+    return ptr;
+}
+
+void operator delete(void* ptr)
+{
+    ucs_memtrack_releasing(ptr);
+    free(ptr);
+}
+
+void *UcxCallback::malloc(size_t size)
+{
+    void *ptr;
+
+    ptr = ::malloc(size);
+    ucs_memtrack_allocated(ptr, size, "io-demo wrapper malloc");
+
+    return ptr;
+}
+
+void UcxCallback::free(void *ptr)
+{
+    ucs_memtrack_releasing(ptr);
+    ::free(ptr);
+}
```

Comment:
1. IMO malloc/free should be public static function in UcxContext
2. can we pass the allocation name as `const char *` to the malloc? MemoryPool already has a name, so we can use it

Reply:
> * IMO malloc/free should be public static function in UcxContext
> * can we pass the allocation name as `const char *` to the malloc? MemoryPool already has a name, so we can use it

done